conf/application.conf

```diff
@@ -208,31 +208,10 @@ go-live-date = "2022-03-01"
 # Generic Feature Flags
 features {
   isPreLaunch = false
-  ukCompanyPrivateBeta = true
+  ukCompanyPrivateBeta = false
   partnershipEnabled = true
 }
 
 # User Specific Feature Flags
-allowedUsers = [
```

Comment:
Will need this here https://github.com/hmrc/app-config-production/blob/main/plastic-packaging-tax-registration-frontend.yaml

Reply:
Can remove allow-list entries in prod Thursday evening, and it will fallback to this confusion, ie allowedUsers=[] ... happy with that?